changelog/Microsoft.DirectoryServices.json

```
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "b3878d32-120c-4469-9ab0-b524a2eb6f79",
+          "ApiChange": "Property",
+          "ChangedApiName": "userFlowApiConnectorConfiguration",
+          "ChangeType": "Addition",
+          "Description": "Added new navigation property **preTokenIssuance** to [userFlowApiConnectorConfiguration](https://docs.microsoft.com/graph/api/resources/userflowapiconnectorconfiguration?view=graph-rest-beta&preserve-view=true) which represents a new step that can be used to enrich tokens in Azure AD B2C user flows.",
+          "Target": "userFlowApiConnectorConfiguration"
+        }
+      ],
+      "Id": "a6f00588-849f-4b2f-b9b7-ca39d6beed75",
+      "Cloud": "prd",
+      "Version": "beta",
+      "CreatedDateTime": "2021-07-21T16:50:00Z",
+      "WorkloadArea": "Identity and access",
+      "SubArea": "Identity and sign-in"
+    },
```

Applying the [changelog guidance](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103153/Changelog).
```suggestion
    {
      "ChangeList": [
        {
          "Id": "b3878d32-120c-4469-9ab0-b524a2eb6f79",
          "ApiChange": "Property",
          "ChangedApiName": "preTokenIssuance",
          "ChangeType": "Addition",
          "Description": "Added the **preTokenIssuance** property to the [userFlowApiConnectorConfiguration](https://docs.microsoft.com/graph/api/resources/userflowapiconnectorconfiguration?view=graph-rest-beta&preserve-view=true) resource. This property represents a new step that can be used to enrich tokens in Azure AD B2C user flows.",
          "Target": "userFlowApiConnectorConfiguration"
        }
      ],
      "Id": "b3878d32-120c-4469-9ab0-b524a2eb6f79",
      "Cloud": "prd",
      "Version": "beta",
      "CreatedDateTime": "2021-08-13T16:50:00Z",
      "WorkloadArea": "Identity and access",
      "SubArea": "Identity and sign-in"
    },
```